main.go

```diff
@@ -9,6 +14,57 @@ var (
 	buildDate string
 )
 
+type AzureBlobxfer struct {
+	StorageAccountKey  string `json:"account_key"`
+	StorageAccountName string `json:"storage_account"`
+	Container          string `json:"container"`
+	Source             string `json:"source"`
+}
+
 func main() {
 	fmt.Printf("Drone Azure Storage Plugin built at %s\n", buildDate)
+	workspace := plugin.Workspace{}
+	vargs := AzureBlobxfer{}
+
+	plugin.Param("workspace", &workspace)
+	plugin.Param("vargs", &vargs)
+	plugin.MustParse()
+
+	if len(vargs.StorageAccountKey) == 0 {
+		return
+	}
+
+	if len(vargs.Container) == 0 {
+		return
+	}
+
+	cmd := command(vargs, workspace)
+	cmd.Env = os.Environ()
+	cmd.Dir = workspace.Path
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+
+	trace(cmd)
+	err := cmd.Run()
```

Comment:
tracing the command will expose the `StorageAccountKey` in the build logs. I would recommend tracing the command without `StorageAccountKey` and then adding `StorageAccountKey` before running the command.

or create two commands, once with `StorageAccountKey` and one without, and trace the command without.